core/src/components/datetime/datetime.ionic.scss

```
@@ -200,3 +200,16 @@
   background-color: globals.$ion-primitives-neutral-100;
   color: globals.$ion-primitives-neutral-1200;
 }
+
+// Calendar / Footer / Action Buttons
+// -----------------------------------
+
+:host .datetime-buttons ion-buttons {
+  display: block;
```

```suggestion
  display: flex;
  flex-flow: column;
  align-items: stretch;
  gap: globals.$ion-space-250;
```

If you want to implement this in a different way that's fine but it looks like there is a gap in Figma between buttons.

---

https://github.com/ionic-team/ionic-framework/pull/30014/commits/0e0641950f1bbc95a15ff0d2214f39c5ea9ed2a8 and https://github.com/ionic-team/ionic-framework/pull/30014/commits/6d3fe984e42fe48037197c8359bf6206b0cdf5d6